__init__.py

```
@@ -54,58 +58,54 @@
                                for option in expand_options(line)]
                     self.intent_matchers[l2].add_intent(intent_name, samples)
 
-    def get_app_aliases(self):
+    def get_app_aliases(self) -> Dict[str, str]:
+        """Fetch application aliases based on desktop files and settings."""
         apps = self.settings.get("user_commands") or {}
         norm = lambda k: k.replace(".desktop", "").replace("-", " ").replace("_", " ").split(".")[-1].title()
```

Ruff

Do not assign a `lambda` expression, use a `def`

Rewrite `norm` as a `def`

(E731)